requirements.txt

```diff
@@ -93,6 +93,7 @@ wasmer==0.4.1             # via -r requirements.txt, fastdiff
 wcwidth==0.2.5            # via -r requirements.txt, prompt-toolkit
 werkzeug==1.0.1           # via -r requirements.txt
 xlrd==1.2.0               # via -r requirements.txt
+algoliasearch-django==1.7.1  # via -r requirements.txt
```

Comment:
In a virtualenv run `pip-compile --allow-unsafe --generate-hashes requirements.txt` ;
make sure additionally u r able to install using pip3 in that env.